### PR TITLE
chore(docs): add Fern Docs Overview page

### PR DESCRIPTION
**Pull request by dannysheridan**

### CODE REVIEW

File: fern/pages/docs/getting-started/docs-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  >
+    Use pre-built or custom React components for a polished look.
+  </Card>
+
+</CardGroup>
+
+## Documentation Types
+
+<Tabs>
+  <Tab title="For Developers">
```
Comment (dsinghvi) [inline fern/pages/docs/getting-started/docs-overview.mdx:64]: it was a bit hard to follow what you mean by Documentation Types -- whats the goal of this section?

Comment (dannysheridan) [reply fern/pages/docs/getting-started/docs-overview.mdx:64]: My goal here is to explain that fern docs support multiple types of content:
- landing page
- guides
- API ref
- changelog

Comment (dannysheridan) [reply fern/pages/docs/getting-started/docs-overview.mdx]: revised in commit [c5fe1f5](https://github.com/fern-api/fern/pull/5500/commits/c5fe1f53fb7310f5541a38f20780543987a2c01d)

**Comment by dannysheridan** [comment]
Page in question: https://fern-preview-2ad7cc94-ffe6-4584-8ec0-cc8aa44b918d.docs.buildwithfern.com/learn/docs/getting-started/overview

File: fern/pages/docs/getting-started/docs-overview.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+title: Fern Docs Overview
+description: Learn about key features of Fern's Docs
+subtitle: A static site generator for developer-focused companies to build and host a beautiful, professional docs site.
+layout: overview
+---
+
+Fern Docs supports multiple content types, including:
```
Comment (chdeskur) [inline fern/pages/docs/getting-started/docs-overview.mdx:8]: What about something like: "Fern Docs provides versatile documentation to support all your content needs, including popular formats like:"
